cmd/kube-proxy/app/server_others.go

```diff
@@ -71,23 +71,40 @@ func newProxyServer(
 		return nil, fmt.Errorf("unable to register configz: %s", err)
 	}
 
-	protocol := utiliptables.ProtocolIpv4
-	if net.ParseIP(config.BindAddress).To4() == nil {
-		klog.V(0).Infof("IPv6 bind address (%s), assume IPv6 operation", config.BindAddress)
-		protocol = utiliptables.ProtocolIpv6
-	}
-
-	var iptInterface utiliptables.Interface
+	// Define IPv4 and IPv6 interfaces.
+	// A nil pointer indicates that the interface is not enabled.
+	var ipt4Interface *utiliptables.Interface
+	var ipt6Interface *utiliptables.Interface
 	var ipvsInterface utilipvs.Interface
+
 	var kernelHandler ipvs.KernelHandler
 	var ipsetInterface utilipset.Interface
+
 	var dbus utildbus.Interface
+	dbus = utildbus.New() // TODO(vllry) does this need to be unique per-iptables interface?
 
 	// Create a iptables utils.
-	execer := exec.New()
+	execer := exec.New() // TODO(vllry) does this need to be unique per-iptables interface?
+
+	// Create iptables interfaces.
+	// Creates an IPv4 and/or IPv6 interface, depending on the types of BindAddresses.
+	for _, bindAddress := range config.BindAddress {
```

Comment:
I think that the behavior will be that the last addresses in config.BindAddress will get the iptXInterface ... don´t remember if we are limiting config.BindAddress to be only IPv4, IPv6 or having one IPv4 and other IPv6 address